website/content/docs/job-specification/template.mdx

```diff
@@ -92,14 +92,34 @@ refer to the [Learn Go Template Syntax][gt_learn] Learn guide.
   One of `source` or `data` must be specified, but not both. This source can
   optionally be fetched using an [`artifact`][artifact] resource. This template
   must exist on the machine prior to starting the task; it is not possible to
-  reference a template inside of a Docker container, for example.
+  reference a template inside a Docker container, for example.
 
 - `splay` `(string: "5s")` - Specifies a random amount of time to wait between
   0 ms and the given splay value before invoking the change mode. This is
   specified using a label suffix like "30s" or "1h", and is often used to
   prevent a thundering herd problem where all task instances restart at the same
   time.
 
+- `wait` `(Code: nil)` - Defines the minimum and maximum amount of time to wait
+  for the Consul cluster to reach a consistent state before rendering a template.
+  This is useful to enable in systems where Consul is experiencing a lot of flapping,
+  because it will reduce the number of times a template is rendered. This configuration is
+  also exposed at the _task level_ to allow overrides per task. This is typically
+  set globally in the client configuration, but can be overridden on per-template
+  basis. Note that, this setting can be overridden by the `client.template.wait_bounds`
+  setting. The `client.template.wait_bounds` setting defines a client level lower
+  and upper bounds for per-template `wait` configuration. If the individual template
+  configuration has a `min` lower than `client.template.wait_bounds.min` or a `max`
+  greater than `client.template.wait_bounds.max`, the bounds will be enforced,
+  and the template `wait` will be adjusted before being sent to `consul-template`.
```

Comment:
This is a little confusing to me. With this wording I'd expect there to be a doc at `/docs/job-specification/task.mdx` as well, which isn't the case.

I'd leave out that "task level" sentence entirely and then reduce the wordiness around the client configuration option:

```suggestion
  because it will reduce the number of times a template is rendered. This setting 
  can be overridden by the [`client.template.wait_bounds`]. If the template
  configuration has a `min` lower than `client.template.wait_bounds.min` or a `max`
  greater than `client.template.wait_bounds.max`, the client's bounds will be enforced
  and the template `wait` will be adjusted before being sent to `consul-template`.
```

(and include the link reference to the client's `wait_bounds` setting at the bottom of the page)